explore/src/main/scala/explore/config/sequence/GmosSequenceTable.scala

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.config.sequence
+
+import cats.syntax.all.*
+import crystal.react.reuse.*
+import explore.Icons
+import explore.components.ui.ExploreStyles
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.all.svg.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.math.Angle
+import lucuma.core.math.Offset
+import lucuma.core.math.Wavelength
+import lucuma.core.model.sequence.*
+import lucuma.react.syntax.*
+import lucuma.react.table.*
+import lucuma.ui.reusability.*
+import lucuma.ui.syntax.all.given
+import lucuma.ui.table.*
+import react.common.Css
+import react.common.ReactFnProps
+import reactST.{tanstackTableCore => raw}
+
+case class GmosSequenceTable(atoms: List[Atom]) extends ReactFnProps(GmosSequenceTable.component)
+
+object GmosSequenceTable:
+  private type Props = GmosSequenceTable
+
+  private val ColDef = ColumnDef[GmosSequenceRow.FutureStep]
+
+  private def drawBracket(rows: Int): VdomElement =
+    svg(^.width   := "1px", ^.height := "15px", ^.overflow.visible)(
+      use(
+        transform := s"scale(1, ${math.pow(rows.toDouble, 1.08)})",
+        xlinkHref := "#bracket"
+      )
+    )
+
+  private val AtomStepsColumnId: ColumnId = ColumnId("atomSteps")
+
+  private val columns: List[ColumnDef[GmosSequenceRow.FutureStep, ?]] =
+    ColDef(AtomStepsColumnId,
+           _.firstOf,
+           header = " ",
+           cell = _.value.map(drawBracket),
+           size = 30.toPx
+    )
+      +: SequenceColumns.gmosColumns(ColDef, _.some)
+
+  private def buildLines(atoms: List[Atom]): List[GmosSequenceRow.FutureStep] =
+    atoms
+      .map(atom =>
+        atom.steps.headOption
+          .map(head =>
+            GmosSequenceRow.FutureStep.fromStep(head, atom.id, atom.steps.length.some.filter(_ > 1))
+          ) ++
+          atom.steps.tail.map(step => GmosSequenceRow.FutureStep.fromStep(step, atom.id, none))
+      )
+      .flatten
+
+  val bracketDef =
+    svg(^.width := "0", ^.height := "0")(
+      defs(
+        path(
+          transform   := "scale(1, 0.28)",
+          fill        := "white", // FIXME Use CSS
```

Comment:
A FIXME comment

Reply:
This was here before, but we should fix it and use Prime colors. I'll address it in another PR.